modules/sts-mode-installing-verifying.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="sts-mode-installing-verifying_{context}"]
+= Verifying the installation
+
+. Connect to the {product-title} cluster.
+
+. Verify that the cluster does not have `root` credentials:
++
+[source,terminal]
+----
+$ oc get secrets -n kube-system aws-creds
+----
++
+The output should look similar to:
++
+[source,terminal]
+----
+Error from server (NotFound): secrets "aws-creds" not found
+----
+
+. Verify that the components are assuming the IAM roles that are specified in the secret manifests, instead of using credentials that are created by the CCO:
```

Comment:
I've always wondered if we should spell out IAM, but it might be AWS is so well known what it means is understood at this point. (Still not by me though.)